capa-controller-role/giantswarm-capa-role.tf

```diff
@@ -7,21 +7,22 @@ locals {
 terraform {
   required_providers {
     aws = {
-      source  = "opentofu/aws"
+      source  = "hashicorp/aws"
```

Comment:
What's the difference between open vs. HashiCorp?

Reply:
Nothing, but interestingly enough OpenTofu shows a warning when using the `opentofu` namespace:

> 
> │ Warning: Additional provider information from registry
> │
> │ The remote registry returned warnings for
> │ registry.opentofu.org/opentofu/aws:
> │ - You are using opentofu/aws instead of hashicorp/aws. This provider is
> │ maintained by HashiCorp and the OpenTofu project only builds the provider
> │ binaries from the source code. Both the opentofu/ and hashicorp/ namespaces
> │ in the OpenTofu Registry contain the OpenTofu-built provider binaries based
> │ on the HashiCorp source code. To avoid configuration errors, we recommend
> │ using the hashicorp/ namespaced providers.

So I followed their advice and switched to the `hashicorp` namespace